lib/piwik/events.rb

```diff
@@ -12,4 +12,40 @@ class Events < ApiModule
       getCategoryFromNameId
     }
   end
-end
\ No newline at end of file
+
+  def category params = {}
+    getCategory(defaults.merge(params))
+  end
+
+  def action params = {}
+    getAction(defaults.merge(params))
+  end
+
+  def name params = {}
+    getName(defaults.merge(params))
+  end
+
+  def action_from_category params = {}
+    getActionFromCategoryId(defaults.merge(params))
```

Comment:
same questions goes for camelCase vs snake_case

Reply:
`getActionFromCategory` is the method from Matomo's API. If you check the [whole file](https://github.com/Monsido/piwik-ruby-api/blob/8d04c7746b28cd5629994e1963fc3c258f1a2c3a/lib/piwik/events.rb) you can see the way the gem is structured.